src/CommandLineUtils/Internal/ValueParsers/EnumParser.cs

```diff
@@ -14,6 +14,14 @@ public EnumParser(Type enumType)
             _enumType = enumType;
         }
 
+        public Type TargetType
+        {
+            get
+            {
+                throw new InvalidOperationException($"{nameof(NullableValueParser)} does not have a target type");
```

Comment:
Shouldn't this just return _enumType? Also, why does it say NullableValueParser?

Reply:
I've made it return `_enumType`.

I chose an error because `Enum`s are special cases and the `EnumParser` never uses `TargetType`.